api/src/main/java/org/openmrs/module/fhir2/api/FhirPatientService.java

```diff
@@ -28,6 +30,10 @@
 	@Override
 	Patient get(@Nonnull String uuid);
 	
+	List<Patient> getByIds(Set<Integer> ids);
```

Comment:
Is there a reason to prefer `Set` over `Collection` here?

Reply:
Yes. To avoid duplicates. `Set` enforces the *set logic* - no duplicates & unordered

Reply:
Well, so where I'm going with this is:

* We've used `Collection` for other similar places
* The `Restrictions.in()` takes a `Collection` rather than a set.
* The de-duplication isn't exactly necessary here (since `SELECT * FROM patients WHERE patient_id IN (1, 2, 3, 1)` has the exact same result as `SELECT * FROM patients WHERE patient_id IN (1, 2, 3)`

Either way, it's not a big deal, just a question.